lib/oxidized/model/awplus.rb

```diff
@@ -0,0 +1,51 @@
+class Awplus < Oxidized::Model
+
+  #Allied Telesis Alliedware Plus Model#
+
+  prompt /^(\r?[\w.@:\/-]+[#>]\s?)$/
+  comment  '! '
+
+  #This shows the system information at the top and comments it out#
+  cmd 'show system' do |cfg|
+    cfg = cfg.each_line.to_a[0...-2]
+
+
+   # Strip system (up)time/ram/free flash.
+    cfg = cfg.reject { |line| line.match /System Status\s*.*/ }
```

Comment:
Have you got any suggestions for how I could do this? 

My google-fu is failing me...

I've actually changed the config in mine from using cfg.reject to cfg.gsub!

This is after suggestions on https://github.com/ytti/oxidized/issues/933

So I now have 

````  
 cmd 'Show System' do |cfg|
    cfg.gsub! /System Status\s*.*/, ''
    cfg.gsub! /RAM\s*:.*/, ''
    cfg.gsub! /Uptime\s*:.*/, ''
    cfg.gsub! /Flash\s*:.*/, ''
    cfg.gsub! /Current software\s*:.*/, ''
    cfg.gsub! /Software version\s*:.*/, ''
    cfg.gsub! /Build date\s*:.*/, ''
````

If you can suggest a way of putting all this on line line, I'm more than happy to confirm if it works.

Thanks,

Matt